Backend/tests/test_user_ctrl.py

```diff
@@ -165,6 +164,93 @@ def test_activate(self):
     isUserActive = 'status' in response.payload.keys() and response.payload['status'] == 'Success'
     self.assertTrue(isUserActive)
 
+  def test_changeUserRole(self):
```

Comment:
consider changing it back to 'student' at the end of the test, to leave the test data intact